webapp-django/crashstats/signature/jinja2/signature/signature_reports.html

```diff
@@ -26,6 +26,8 @@
                     {% if crash[column] %}
                         {% if column == 'date' %}
                             {{ crash[column] | human_readable_iso_date }}
+                        {% elif column == 'install_time' %}
+                            {{ crash[column] | timestamp_to_date }}
```

Comment:
The value is not always there. How did jinja not get an error here?
<img width="1616" alt="screen shot 2016-04-18 at 10 47 46 am" src="https://cloud.githubusercontent.com/assets/26739/14608531/1bfa2164-0554-11e6-82cc-7a8c69e530d7.png">

If `{{ crash[column] }}` becomes an empty string, I suspect that's because we're lucky about how jinja handles empty values. 
Either way, a safer approach (and more transparent) would be something like this:
```
{% elif column == 'install_time' %}
  {% if crash[column] %}
    {{ crash[column] | timestamp_to_date }}
  {% endif %}
{% else %}
```